### DISCUSSION TITLE
SP-Sales-API: How to get the sales data of each updated product?

### DISCUSSION THREAD

**Discussion by tuannt1531** [amzn/selling-partner-api-models#3497] [General]
Hi everybody,
- we are using Sales API to get data for our product, we get data periodically every day, however, there are cases where the data in the past has changed and updated, so the data that we took earlier will no longer be accurate
- Currently, our solution is to periodically get the data of the last 4 months to ensure that the old data is updated correctly, but with these we have to request a lot and sometimes get blocked due to the API request limit
- is there any solution or API that only needs to get the data of the changed product codes in the past without getting the entire data of all the product codes?
Thanks a lot!

**Comment by phuongph1997** [comment]
hi guy, i have the same issue, have you find the result???

**Reply by phuongph1997** [reply]
i want to know, when product update, title, image, price,.vv,  what event will trigger ??
